### DISCUSSION TITLE
The Collector is not receiving data.

### DISCUSSION THREAD

**Discussion by murphypa271327** [cisagov/LME#87] [FAQ] [answered]
I'm getting the following error and wanted to see if I'm missing something. My Elk server works great. I included the Collector in my test batch and it shows up in the program and I get all kinds of data! But, none of the other servers and reaching it! I show zero subscriptions and each server has the same error - 
The forwarder is having a problem communicating with subscription manager at address https://example.com:5985/wsman/SubscriptionManager/WEC.  Error code is 12175 and Error Message is <f:WSManFault xmlns:f="http://schemas.microsoft.com/wbem/wsman/1/wsmanfault" Code="12175" Machine="client.com><f:Message>The server certificate on the destination computer (example.com:5985) has the following errors:       
Encountered an internal error in the SSL library.   </f:Message></f:WSManFault>.

**Comment by llwaterhouse** [comment, marked as answer]
Hello, 

Thank you for joining in on our Github Discussions.  This type of question is best for our Issues area, https://github.com/cisagov/LME/issues, because our template guides you to do some initial troubleshooting and also requests information that we may be able to use to identify what is going on.

Please re-submit your ticket with the information so that we can better help you.
Thank you.
